lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -1648,6 +1648,13 @@ bool StructuralEquivalenceContext::CheckKindSpecificEquivalence(
     }
   } else if (FunctionDecl *FD1 = dyn_cast<FunctionDecl>(D1)) {
     if (FunctionDecl *FD2 = dyn_cast<FunctionDecl>(D2)) {
+      if (FD1->isOverloadedOperator()) {
+        if (!FD2->isOverloadedOperator())
+          return false;
+        if (getOperatorSpelling(FD1->getOverloadedOperator()) !=
+            getOperatorSpelling(FD2->getOverloadedOperator()))
```

Comment:
Why not compare the value from `getOverloadedOperator` directly (instead of spelling)?

Reply:
Good point.